docs/deploy-cloudstate-cluster.adoc

```diff
@@ -0,0 +1,152 @@
+
+
+[[signup-cloudstate-account]]
+=== Sign-up Cloudstate account
+
+To deploy your service on Lightbend Cloudstate host platform, you will need to sign up an account. With your Cloudstate account, you will be granted a Kubernetes cluster on the cloud. This cluster is managed by Lightbend Cloudstate platform. It is customized to host services created using the https://github.com/cloudstateio/cloudstate[Cloudstate library].
+
+Instructions: https://docs.lbcs.dev/getting-started/lightbend-account.html[Using a Cloudstate account]
+
+
+=== Install Cloudstate CLI
+
+`csctl` is the command line tool that you will use to deploy application on your Cloudstate cluster. Instructions: https://docs.lbcs.dev/getting-started/set-up-development-env.html#_cloudstate_cli[Install Cloudstate CLI]
+
+The `csctl` minimum version required is 0.0.9 which you can verify by:
+
+[source,shell]
+----
+$ csctl version
+0.0.9
+----
+
+
+=== Creating a Cloudstate project
+
+A Cloudstate project is a space within your Cloudstate cluster. In this space, you will deploy your Cloudstate-aware application. 
+
+In this step we use the Cloudstate CLI `csctl` to:
+
+* Login to your Cloudstate cluster (which you got when signing-up for a <<signup-cloudstate-account,Cloudstate account>>)
+* Create a Cloudstate project
+* Deploy your services in the project
+
+More details on `csctl` is available at https://docs.lbcs.dev/cli/index.html[csctl reference documentation].
+
+
+[source,shell]
+----
+#===STEP1: connect csctl to your Cloudstate account
+$ csctl auth login
+
+#===STEP2: create a new Cloudstate project
+# Syntax: csctl project new  "project-description" "ignored"
+# NOTE: no space, only lower case
+$ csctl project new chat-application-v2 "Chat room app built with Cloudstate services"
+
+Created project b43197f4-5c75-4bac-af18-8b2be7460696.
+To make this the currently active project, run: 'csctl config set project b43197f4-5c75-4bac-af18-8b2be7460696'
+
+#===STEP3: set the new Cloudstate project as "current"
+$ csctl config set project b43197f4-5c75-4bac-af18-8b2be7460696
+
+#===STEP3: Verify project created OK
+$ csctl project list
+
+  NAME                  DESCRIPTION                                    STATUS    ID
+* chat-application      Chat room app built with Cloudstate services   pending   b43197f4-5c75-4bac-af18-8b2be7460696
+----
+
+IMPORTANT:: A newly created project is in `pending` status, ie. you cannot yet deploy on your Cloudstate cluster. 
+Once the project is activated by Cloudstate administrator (within a short moment). Its status will change to `active`, at which point, you can begin to deploy your services. In case it takes long time for your project to get activated, you can https://docs.lbcs.dev/getting-started/create-project.html#_project_status[[notify Cloudstate admin]
```

Comment:
```suggestion
Once the project is activated by a Cloudstate administrator (within a short moment). Its status will change to `active`, at which point, you can begin to deploy your services. In case it takes long time for your project to get activated, you can https://docs.lbcs.dev/getting-started/create-project.html#_project_status[[notify Cloudstate admin]
```

Reply:
`within a short moment` usually :-)